2024.md

```diff
@@ -1,9 +1,10 @@
 # 2024 Blogs
 
-## January (2)
+## January (3)
 * [Coding Authentication? Stop now and let Choreo manage it.](https://dev.to/suvink/coding-authentication-stop-now-and-let-choreo-manage-it-4ofa) by [Suvin Nimnaka](https://dev.to/suvink)
 * [CRYSTALS Kyber : The Key to Post-Quantum Encryption](https://medium.com/@hwupathum/crystals-kyber-the-key-to-post-quantum-encryption-3154b305e7bd) by [Udara Pathum](https://medium.com/@hwupathum)
-
+* [Elevating Security with Asgardeo’s “client-request” Token Binding Type](https://vivekvinushanth.medium.com/elevating-security-with-asgardeos-client-request-token-binding-type-ca649fe27890) by Vivekvinushanth Christopher
```

Comment:
Hope omitting adding author's profile link is intentional :)

Reply:
It was. But updated now.